x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents_logic.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kea, MakeLogicType } from 'kea';
+
+export interface DocumentsLogicValues {
+  isDocumentCreationOpen: boolean;
+}
+
+export interface DocumentsLogicActions {
+  closeDocumentCreation(): void;
+  openDocumentCreation(): void;
+}
+
+type DocumnetsLogicType = MakeLogicType<DocumentsLogicValues, DocumentsLogicActions>;
```

Comment:
[nit] super minor spelling
```suggestion
type DocumentsLogicType = MakeLogicType<DocumentsLogicValues, DocumentsLogicActions>;
```

Reply:
🤦